experimental/Schemes/BlowupMorphism.jl

```diff
@@ -3,6 +3,36 @@ export center
 export exceptional_divisor
 export projection
 
+@doc raw"""
+    AbsDesingMor{
```

Comment:
```suggestion
    AbsDesingMor{
```
Please consider renaming to `AbsDesingularizationMor`.